oshmem/mca/sshmem/ucx/sshmem_ucx_module.c

```diff
@@ -192,9 +192,9 @@ segment_create(map_segment_t *ds_buf,
             SSHMEM_WARN("Failed to allocate DEVICE_NIC_MEM, "
                         "fallback to host memory");
         }
-#else
-        SSHMEM_WARN("UCX version is too old and won't support "
-                    "SHMEM_HINT_DEVICE_NIC_MEM hint, fallback to host memory");
+#else 
+        SSHMEM_WARN("UCX is not supporting SHMEM_HINT_DEVICE_NIC_MEM "
```

Comment:
does not support

Reply:
`SSHMEM_VERBOSE(1, "DEVICE_NIC_MEM hint ignored since UCX does not support MEMORY_TYPE_RDMA")`